packages/next/src/build/entries.ts

```diff
@@ -422,7 +422,7 @@ export function getEdgeServerEntry(opts: {
   }
 
   return {
-    import: `next-edge-ssr-loader?${stringify(loaderParams)}!`,
+    import: `next-edge-ssr-loader?${JSON.stringify(loaderParams)}!`,
```

Comment:
These needs to be querystring stringified as the parsing doesn't use JSON.

Reply:
@timneutkens might affect how webpack-internal references for this loader look like